### ISSUE TITLE
Can't Send ETH using Mist...

### ISSUE THREAD

**Issue by frank-optinum** [ethereum/mist#3112]
- [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.0.0`
OS & Version: windows/linux/osx
Node version: `geth 0.0.0` 
Number of blocks synchronized: 0
```

**Comment by nightflight** [comment]
If that helps, I am experiencing the same problem with Ethereum Wallet 0.9.1 on Win 7 Pro 64-bit. The wallet is in sync and I I have received funds on my Main account (Etherbase) but when sending to external destinations (tried Shapeshift and Coinbase so far) the transaction gets stuck on 0 of 12 confirmations, until I get notified that the transaction could not be successfully completed.

**Comment by Rh0msfer** [comment]
I've had this problem with 0.9.1 and 0.9.0 both on Mac OS X and Win 10 on 3 different machines. I've tried updating geth to no avail. Also cleared out the cache and tried again... Now trying 8.x.x versions. This is very frustrating. 

Any ideas anyone?

**Comment by EthWoo** [comment]
I have the same issue and all my Ethereums are like frozen in the account.
Using the Ethereum Wallet Version 0.9.2 for Windows. 
I am also fully synched to the blockchain

When I try sending a transaction, it pops up with 0 of 12 confirmations ans dissapears after about 10 minutes agains. There are no logs of it on Etherscan as well.

How can this be solved?
Thanks

**Comment by analogboybcn** [comment]
I have the same problem.

I have sent some Mana to the Decentraland contract this morning in order to reserve some land there. Then my transaction get stucked. 
So, I contacted the Decentraland team and they were kind enough to rapidly solve the issue. My mana was reserved to some land in a District. 

But the problem I'm facing now is I'm unable to send ether or any ERC20 token outside this wallet. 
I have tried infinite things, and lost several hours trying to solve the issue with no luck.

Some help will be much appreciated.

**Comment by woooops** [comment]
I have the same problem, trying to send out eth but can't, fully synched and over 8 peers.

**Comment by thebeephaha** [comment]
I am having this issue as well:

![transaction](https://user-images.githubusercontent.com/32210641/32265256-805decb6-be9f-11e7-8ead-f27a7324e234.JPG)
![transaction2](https://user-images.githubusercontent.com/32210641/32265257-80788710-be9f-11e7-9d8a-eca3f47973a3.JPG)
![transaction3](https://user-images.githubusercontent.com/32210641/32265258-808e1562-be9f-11e7-848f-9b7f289c1ac3.JPG)

https://etherscan.io/tx/0x21914deb63cbb9afc98b53872425d6bf215c13cbd013e37413d4dc6938ec1be0 ("Sorry, We are unable to locate the Transaction entry")

**Comment by lefiguier** [comment]
Hello :)
I have the same pb could you help me please..
I bought yesterday ETH on coinbase and transferred directly to my MIST ETH account, I can see I have the ETH on etherscan, that is well on my account but I cannot see it on the balance and cannot send ETH, 
what I can do please

**Comment by analogboybcn** [comment]
Hi lefiguier. The problem was solved several days ago.

Seeing that you recently bought some Ether on Coinbase (a day ago) I'm wondering that maybe your Mist wallet needs to end the sync with the Ethereum network.

FYI: Some weeks ago I bought a new computer. My mist wallet on this computer needed 5 entire days to sync to the Ethereum network and show my money. Meantime the balance on the wallet was 0.

**Comment by lefiguier** [comment]
Hi Analogboybcn,
Thank you for your answer I was really embarrassed...
So you think, that if my wallet is still creating is it normal?

**Comment by lefiguier** [comment]
<img width="836" alt="screen shot 2017-11-14 at 17 08 29" src="https://user-images.githubusercontent.com/33655827/32793664-86d1735a-c95e-11e7-9d67-268d19ff91e8.png">

**Comment by lefiguier** [comment]
<img width="815" alt="screen shot 2017-11-14 at 17 09 54" src="https://user-images.githubusercontent.com/33655827/32793724-9e90e002-c95e-11e7-8df4-de935b0eea04.png">

**Comment by analogboybcn** [comment]
Yes. You need to wait until the sync (which may takes some days).

As you can see in the lower right you have 0 blocks downloaded.
my mist wallet shows  4494913 blocks downloaded (and I have not synced since a week ago, so there must some blocks left)

<img width="147" alt="screen shot 2017-11-14 at 19 03 49" src="https://user-images.githubusercontent.com/13234834/32796341-c421dd70-c96e-11e7-9fb0-2929f12a228e.png">

Dont' be affraid. You just need to have a little patience during the days your wallet syncs to the network.
Ass soon as you are up to date, you will see you real balance. ;)

**Comment by lefiguier** [comment]
Ok thank you :)
but is it normal that I have 0.000.000 blocks dowloaded since yesterday? it will arrive all in one shot?

**Comment by analogboybcn** [comment]
Whoop!
It's not normal. It should be downloading blocks and you should see the new blocks added to the list.

Start Mist again and don't click in the "launch application" button. I have found that sometimes my mist app preferred to sync without the program open (but that was in previous mist versions)... 

Did you see if it connects to some peers connected?

I'm not an expert dev at all... But maybe if you share your log file someone is able to better help.
Your LOG file can be located by going to **File / Backup / Accounts** then open the **Geth** folder and then **chaindata**.

**Comment by lefiguier** [comment]
Ok thank you I will try to ...
You mean I have to launch Mist (without click on "Launch application") and just let it works?
No it does not print peers on the bottom, 0 BLOCKS ; 0 PEERS ... 

_Do you know someone able to help on this kind of matter, because there is not other way that GitHub or forums to find help..._

Thank you again for your answer

**Comment by felcher69** [comment]
yeah. I have same problem. network is synched, & it just sits there wih 0 of 12 confirmations. I have tried sending maybe 10 times now
